scripts/common.py

```diff
@@ -418,6 +418,22 @@ def check_prs_for_current_release_are_released(repo):
                 'released before release summary generation.')
 
 
+def normalize_windows_style_path(file_path):
```

Comment:
If possible, don't implement this function at all. Use os.path methods instead to construct a path correctly regardless of the specific OS.

Reply:
Just tried the build-in one on Windows and it is not doing what I intended to do. 

```
In [4]: import os

In [5]: a = os.path.join('test', 'path')

In [6]: a
Out[6]: 'test\\path'

In [7]: os.path.normpath(a)
Out[7]: 'test\\path'
```
The one I defined is intended to transfer the Windows-style path to Unix-style (as the paths in compiled HTML files are always in Unix-style). Maybe I should rename this to `transfer_windows_style_path_to_unix_style` for good.

Reply:
Hm, could you explain what you mean by "paths in compiled HTML files"? That sounds like URLs, so is it possible that what you're talking about here is URLs vs directory tree filepaths? If so, I would suggest making that distinction instead of unix vs windows.